sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -390,10 +393,13 @@ private async Task RunBatchReceiveLoopAsync(CancellationToken cancellationToken)
                         }
                     }
 
-                    IReadOnlyList<ServiceBusReceivedMessage> messages =
-                        await receiver.ReceiveMessagesAsync(
-                            _serviceBusOptions.MaxMessageBatchSize,
-                            cancellationToken: cancellationToken).AwaitWithCancellation(cancellationToken);
```

Comment:
AwaitWithCancellation is no longer needed as underlying SDK handles cancelling in progress receives now.